app/services/achiever/course/template.rb

```diff
@@ -66,6 +67,7 @@ def self.from_resource(resource, activity)
       t.workstream = resource["Template.Workstream"]
 
       t.always_on = activity&.always_on || false
+      t.is_new_course = activity.created_at > 3.months.ago
```

Comment:
The ticket is wanting to have this tag displayed whenever a CPD is created or reauthored.

This looks like its using the `created_at` timestamp from the activity record in our database. Does this mean that the tag will only show three months after its been created in our database? I'm wondering if there is some `Template.CreatedAt` or similar timestamp that gets provided from Dynamics that we could make use of instead. I'm also wondering if a `created_at` timestamp is the correct thing to use, or whether we should instead be making use of an `updated_at` or even an `authored_at` like timestamp... I thing that would probably depend on what reauthoring looks like on the dynamics side though.